jgrapht-core/src/main/java/org/jgrapht/alg/MaximumWeightBipartiteMatching.java

```diff
@@ -0,0 +1,264 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2012, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* -------------------------
+ * MaximumWeightBipartiteMatching
+ * -------------------------
+ * (C) Copyright 2015, by Graeme Ahokas and Contributors.
+ *
+ * Original Author:  Graeme Ahokas
+ * Contributor(s):
+ *
+ * Changes
+ * -------
+ * 30-Sep-2015 : Initial revision (GA);
+ *
+ */
+
+
+
+/* This class is finds a maximum weight matching of a simple undirected 
```

Comment:
Move this to be a Javadoc comment (i.e. starting with /**) right before the class definition.  Also typo:  "This class is finds".